node/cn/handler.go

```diff
@@ -465,14 +465,24 @@ func (pm *ProtocolManager) handle(p Peer) error {
 }
 
 func (pm *ProtocolManager) processMsg(msgCh <-chan p2p.Msg, p Peer, addr common.Address, errCh chan<- error) {
-	for msg := range msgCh {
-		if err := pm.handleMsg(p, addr, msg); err != nil {
-			p.GetP2PPeer().Log().Error("ProtocolManager failed to handle message", "msg", msg, "err", err)
-			errCh <- err
-			return
+	_, fakeF := pm.fetcher.(*fetcher.FakeFetcher)
+	_, fakeD := pm.downloader.(*downloader.FakeDownloader)
+	if fakeD || fakeF {
+		p.GetP2PPeer().Log().Warn("ProtocolManager does not handle p2p messages", "fakeFetcher", fakeF, "fakeDownloader", fakeD)
```

Comment:
Just one question.. Can I ask why the warning msg is logged when it is a fake module? According to the description, it seems to be a normal operation. Or, is it a wrong behavior?

Reply:
I thought discarding messages without handling is a very unexpected behavior of "handler". Because fake modules are introduced only for KES service nodes, I wanna inform the node is using fake modules and a handler is working in unexpected ways to node operators.

Reply:
Oh, using the fake module is not desirable behavior to non KES service nodes. Understood :)